src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -120,8 +120,40 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, string meth
                         return true;
                     }
 
+                case "Runtime.exceptionThrown":
+                    {
+                        if (!GetContext(sessionId).IsRuntimeReady)
+                        {
+                            string exceptionError = args?["exceptionDetails"]?["exception"]?["value"]?.Value<string>();
+                            if (exceptionError == "pause_on_uncaught")
```

Comment:
Can you add const fields for the two strings?

Reply:
Yes